src/cljs/replumb/repl.cljs

```diff
@@ -207,22 +231,25 @@
   Read-file-fn! is an async 2-arity function with signature [file-path
   src-cb] where src-cb is itself a function (fn [source] ...) that needs
   to be called with the full source of the library (as string)."
-  [verbose? src-paths read-file-fn!]
```

Comment:
Please change the above doc like in `make-js-eval-fn` where you specify which options are used